### ISSUE TITLE
Activity 1.3 task 5 issue

### ISSUE THREAD

**Issue by ianfetterman** [STA518-02-Fall20/community#12]
I am having trouble with task 5 printing the decoded message. I went through each step of top_secret and feel as though my vector is correct but it is blank when I run the code.

![image](https://user-images.githubusercontent.com/70541521/93090841-579c0e00-f66b-11ea-9ff2-7b45e20cf1d8.png)

**Comment by ianfetterman** [comment]
Here are my first few numbers printed in my final top_secret vector

![image](https://user-images.githubusercontent.com/70541521/93091348-f1fc5180-f66b-11ea-898c-a4e1c4861fc2.png)

**Comment by dykesb** [comment]
What did you do to fix this?

**Comment by ianfetterman** [comment]
I just went back through the beginning steps and fixed a few minor errors with my variable names and played around with some things until it finally read the Carroll poem correctly. I had an issue with the --names(my_symbols) = "Symbol"-- line of code as well as a few other things I had to change back to how they had been. Some things that I changed while trying to troubleshoot were actually incorrect.
